Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openbuildservice/base
```

Comment:
I think you will need a FROM openbuildservice/base:423 here.

Reply:
This will pull just latest...

Reply:
I don't see it: https://hub.docker.com/r/openbuildservice/base/tags/ 
There is only the 423 tag.

Reply:
Man docker tags are hard to understand. They are there now sorry...